docs/network.md

```diff
@@ -0,0 +1,55 @@
+# 10. Message transportation layers.
+
+This section describes different protocols of sending messages currently used in `Near`
+
+## 10.1 Messages between Actors.
+
+`Near` is build on `Actix`'s `actor` framework. (https://actix.rs/book/actix/sec-2-actor.html)
+Usually each actor runs on its own dedicated thread.
+Only messages implementing `actix::Message`, can be sent using between threads.
+Comment on lines +7 to +9
+
+ bowenwang1996 4 days ago Member
+
+We should not leak implementation details into the spec
+
+	Reply…
+Resolve conversation
+
+On example of such message is `PeersRequest` :
```

Comment:
I'd just mention, that people should look for 'impl actix::Message for XX' to see which messages can be passed such way